cmd/bucket-handlers.go

```diff
@@ -707,6 +748,17 @@ func (api objectAPIHandlers) PutBucketHandler(w http.ResponseWriter, r *http.Req
 	vars := mux.Vars(r)
 	bucket := vars["bucket"]
 
+	// Get PanFS path for bucket only if Minio is running in PanFS Gateway mode
+	// If PanFS path provided in any other mode - just ignore it
+	panfsBucketPath := ""
```

Comment:
Can we set `panfsBucketPath` as `globalPanFSDefaultBucketPath` here?

Reply:
I'm not sure as the value here should be set only for PanFS gateway server mode. In the same time, as I updated `MakeBucketOptions` structure the value for `PanFSBucketPath` must be set for any backend (so it is empty for all backends except PanFS).

Actually, this can be refactored to your version - in that case I can set global value for `panfsBucketPath` by default but only assign it to `MakeBucketOptions` structure (in line 832) in case of `PanFS` backend. But that seems pretty similar for me to the existing variant

Reply:
Let's leave it as it is